docs/workflow_solutions/using_anaconda.md

```diff
@@ -282,6 +282,94 @@ Now we can be sure that the correct versions of the software will be installed o
+
+1. github "name".
+2. gitlab "name".
+
+Remember to replace name with name of Anaconda package.
+
+When we have a complete list of Anaconda packages and Channels, then we can create an environment from scratch with all the dependencies included. For Anaconda packages, add one line to dependencies for each software. For Pypi packages add - pip: under dependencies. Then under - pip:add `==` to pin the version, see below.
+
+```yaml
+name: test-env
+channels:
+  - anaconda
+  - conda-forge
+  - pip
+dependencies:
+  - matplotlib=3.8.4 #pinned version for conda
+  - python=3.10.4
+  - pip
+  - pip:
+  - numpy==1.26.4 #pinned version for pip
+  - http://insert_package_link_here #for url links
+
+```
+
+ For git repos, add them under `- pip:` based on examples [here](https://pip.pypa.io/en/stable/cli/pip_install/#examples). See the section [Replicability versus Portability](#replicability-versus-portability) for more information.
+
+The above configuration is only for illustration purposes, to show how channels and dependencies can be used. It is best to install all of your packages from conda channels, to avoid your environment breaking. Only packages that are unavailable via conda, should be installed via pip. If you run into challenges please [contact us](../index.md#contact-us).
+
+##### Key Things To Remember
+
+1. Exploring Package Documentation: For each package, check the documentation to understand its features, version history, and compatibility. Documentation can often be found on the Anaconda Cloud package page under the "Documentation" or "Homepage" link shared above in this tutorial.
+
+1. Regularly update your environment file to manage dependencies and maintain compatible software environments.
+
+1. Verify Package Version and Maintenance: Ensure you are getting the latest version of the package that is compatible with your environment. Verify that the package is actively maintained by checking the source repository (e.g., GitHub, GitLab). Look for recent commits, releases, and issue resolutions. The concepts of version pinning and semantic versioning, explain this in detail.
+
+##### Version Pinning
+
+Version pinning in Anaconda environments involves specifying exact versions of packages to ensure consistency and compatibility. This practice is crucial for reproducibility, as it allows environments to be reproduced exactly, a critical component in research and collaborative projects. Version pinning also aids stability, by preventing unexpected changes that could break your environment, code or analysis. This practice also maintains compatibility between different packages that rely on specific dependencies. To implement version pinning, you can create a YAML file that lists the exact versions of all installed packages or specify versions directly when [creating](#create-an-environment) or updating environments using Conda commands.
+
+##### Semantic Versioning
```

Comment:
Provide a link to https://semver.org somewhere in the text. It's about as official a standard as exists. Maybe some day it will become an RFC or ISO standard. We can only hope!